manifests/on-prem/keepalived.conf.tmpl

```diff
@@ -25,4 +26,5 @@
     virtual_ipaddress {
         {{ .Cluster.APIVIP }}/{{ .Cluster.VIPNetmask }}
```

Comment:
Most of the magic actually happened in runtimecfg: https://github.com/openshift/baremetal-runtimecfg/pull/176

The way we get multiple VIPs out of this small change is the range call on line 7. We iterate a new list object provided by runtimecfg that includes both VIPs. It has the exact same structure for each item as the original runtimecfg data structure so that's why the templates didn't have to change much.

Reply:
Ahhh ok. This is clear now.